polaris/hub/client.py

```diff
@@ -866,12 +869,15 @@ def evaluate_competition(
              A `BenchmarkResults` object.
```

Comment:
This doesn't return a `BenchmarkResults` anymore right? Just the JSON payload from the hub? We could turn it into one, but I don't think we need to. It's probably fine to just say this returns a 201 response on success or other error response from the hub.

Reply:
Nice catch!